src/components/content/order/OrderSubmit.tsx

```diff
@@ -201,6 +201,25 @@ function OrderSubmit(props: OrderSubmitProps): JSX.Element {
         .concat('&serviceName=', props.name)
         .concat('&latestVersion=', props.version);
 
+    function onclick() {

```

Comment:
As far as i know, these API changes are not yet final. Xinsui is still working on it.

Reply:
I had a discussion with Xinsui just now, he is working, it has no effect on my work content